test/integration/conflict_resolution.js

```diff
@@ -150,6 +150,145 @@ describe('Conflict resolution', () => {
     })
   })
 
+  const afterFullSyncBothTreeDeepEqual = async (expected) => {
+    await helpers.local.scan()
+    await helpers.remote.pullChanges()
+    await helpers.syncAll()
+    await helpers.local.scan()
+    await helpers.remote.pullChanges()
+    await helpers.syncAll()
+    should(await helpers.trees()).deepEqual({remote: expected, local: expected})
```

Comment:
Looks like putting should.js assertions into non-test functions makes stacktraces useless... :sob:

Reply:
I refactored the test helpers to extract the assertions back to the test functions.